### PR TITLE
Fixed to support Big Sur

### PR DESCRIPTION
**Pull request by bergdahl**
Please check the README for more information on this change.

### CODE REVIEW

**Comment by lorol** [comment]
@bergdahl 
Thank for submitting
I am not a fancy hygienic programmer. I am not a programmer at all, but I think on readme PR change you suggested me to merge, you (actually I may do the same mistake too) should not deviate the downloading of product of same work from somewhere else i.e your repository ... anyway.

**Comment by bergdahl** [comment]
Oh, sorry for the confusion - I only ment for you to merge the code changes and make a new release for download. 
The changes for the README is just a note for for my friends that needed this fix fast.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+# Note
+
+This is just a quick and fast hack to get this plugin to work under macOS Big Sur. It fixes the following error.
+```
+ValueError: dlsym(RTLD_DEFAULT, kIOMasterPortDefault): symbol not found
+```
+
+Download the plugin at [https://github.com/bergdahl/arduino-esp32fs-plugin/releases/tag/2.0.6.BigSur](https://github.com/bergdahl/arduino-esp32fs-plugin/releases/tag/2.0.6.BigSur) and follow the original instructions below.
```
Comment (lorol) [inline README.md:8]: Please remove these lines and resubmit

**Review by lorol — CHANGES_REQUESTED** [review]
Readme.md
Line#8

**Comment by bergdahl** [comment]
Reverted README changes

**Review by lorol — APPROVED** [review]

**Comment by lorol** [comment]
The following error was fixed by @bergdahl 
```
ValueError: dlsym(RTLD_DEFAULT, kIOMasterPortDefault): symbol not found
```
## Changes
Changed the call to `esptool` to use the command `python3 esptool.py` instead as
newer versions of the `esptool` tool seems to be Python only. Python version 2 is obsolete but still default on macOS, which is why I used `python3` to call the script.

If you done an in-place upgrade of macOS you also need to upgrade to the latest version of the `pyserial` package as well;
```
pip3 install --upgrade pyserial
```

In case you use newer boards like the ESP32S2, you may also need to upgrade the `esptool.py` file in the board support folder. You can find the existong file under a folder named like `/Users/ -Your user Name- /Library/Arduino15/packages/esp32/tools/esptool_py/2.6.1`. Just replace esptool.py with the latest one. 

You can find the latest version of `esptool` here - [https://github.com/espressif/esptool](https://github.com/espressif/esptool]).

**Pull request merged**